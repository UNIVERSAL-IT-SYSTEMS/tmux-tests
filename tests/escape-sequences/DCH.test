#!/bin/sh

S=$(mktemp)
trap 'rm -f $S' 0 1 15

cat <<EOF >$S
printf "\033[Pa"
printf "\n\033[0Pb"
printf "\n\033[1Pc"
printf "\n\033[2Pd"
printf "\na\033[Pa"
printf "\nb\033[0Pb"
printf "\nc\033[1Pc"
printf "\nd\033[2Pd"
printf "\nabcdef\010\010\010\010\033[P"
printf "\nabcdef\010\010\010\010\033[0P"
printf "\nabcdef\010\010\010\010\033[1P"
printf "\nabcdef\010\010\010\010\033[2P"

$TMUX_L wait -S tests
$TMUX_L wait never
EOF

$TMUX_L new -d -x 80 -y 25 -- sh $S
$TMUX_L wait tests\; capturep -ep -S- -E-
