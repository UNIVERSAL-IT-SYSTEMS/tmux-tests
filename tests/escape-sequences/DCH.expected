a
b
c
d
aa
bb
cc
dd
abdef
abdef
abdef
abef












